app/components/create-player-form.js

```diff
@@ -0,0 +1,44 @@
+import Component from '@glimmer/component';
+import { action, computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { dropTask } from 'ember-concurrency';
+
+export default class CreatePlayerFormComponent extends Component {
+  @service store;
+
+  @tracked
+  nickname = '';
+
+  @computed('nickname', 'args.createMode')
+  get nicknameError() {
+    if (this.args.createMode && !this.nickname.length)
+      return "Nickname can't be blank!";
```

Comment:
I know that this isn't a big app,  but on our projects, we try to use internalization whenever possible
https://github.com/ember-intl/ember-intl. You can do it here or create issue to remember that and do it in seperate PR

Reply:
also, it may be worth considering https://github.com/poteto/ember-changeset  for validations